From 6719e94e4065a1d8e5052c5234debd3dc9b0b594 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 16 Sep 2018 11:54:37 +0300
Subject: [PATCH] release: print release repos list

This helps us in maintaining only a single list of emebedded
repositories for release.
All other s/w components (e.g. jenkins jobs, etc.) will use the
print-release-repos target to get the repos list.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 9cd20f6..81a0070 100644
--- a/Makefile
+++ b/Makefile
@@ -43,6 +43,8 @@
 	@echo '  goya-release         - Prepare the release tag and push it upstream'
 	@echo '                           usually, the version string would be something'
 	@echo '                             like: x.y.z'
+	@echo '  print-release-repos  - Print the list of embedded repositories used for'
+	@echo '                           release'
 	@echo
 	@echo 'Testing:'
 	@echo '  goya-<board>-test    - Run uboot test on <board> e.g. onix, pldm, asic'
@@ -69,6 +71,10 @@
 RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader btl
 RELEASE_TAG_POINTER := latest-embedded-release
 
+.PHONY: print-release-repos
+print-release-repos:
+	@echo $(RELEASE_REPOS)
+
 .PHONY: gaudi-release goya-release
 gaudi-release goya-release: hl-check-release
 	$(eval HL_FULL_VER=hl-$(patsubst %-release,%,$@)-$(HL_RELEASE_VER))
